<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamePlayWidget</class>
 <widget class="QWidget" name="GamePlayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="GamePlayCentralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>322</width>
     <height>402</height>
    </rect>
   </property>
   <widget class="QPushButton" name="exitGameBtn">
    <property name="geometry">
     <rect>
      <x>177</x>
      <y>10</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restartGameBtn">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>10</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Заново</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pauseGameBtn">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>10</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Пауза</string>
    </property>
   </widget>
   <widget class="QLabel" name="timerGame">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>17:14</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
